doc/rdisc.xml

```diff
@@ -61,22 +61,22 @@ xml:id="man.rdisc">
     <command>rdisc</command> can avoid waiting for routers to
     announce themselves by sending out a few ROUTER_SOLICITATION
     messages to the ALL_ROUTERS (224.0.0.2) multicast address (or
-    <emphasis remap="I">send_address</emphasis> provided it is
-    given) when it is started.</para>
+    <emphasis remap="I">send_address</emphasis> if it is provided)
+    when it is started.</para>
     <para>A timer is associated with each router address and the
     address will no longer be considered for inclusion in the the
     routing tables if the timer expires before a new
     <emphasis remap="B">advertise</emphasis> message is received
     from the router. The address will also be excluded from
-    consideration if the host receives an
-    <emphasis remap="B">advertise</emphasis> message with the
-    preference being maximally negative.</para>
+    consideration if the received
+    <emphasis remap="B">advertise</emphasis> message has a
+    preference of maximum negative.</para>
     <para>Server side of router discovery protocol is supported by
     Cisco IOS and by any more or less complete UNIX routing daemon,
-    f.e
-    <emphasis remap="B">gated</emphasis>. Or,
-    <command>rdisc</command> can act as responder, if compiled with
-    ENABLE_RDISC_SERVER.</para>
+    e.g.
```

Comment:
Rather than shorthand of 'exempli gratia' how about 'For example'.

Reply:
I consider *e.g.* as quite common abbreviation, but *for example* obviously ok as well.

Reply:
Me, too. So consider this done.